src/main/java/com/softserveinc/ita/pageobjects/HomePage.java

```diff
@@ -24,7 +24,7 @@ public ProductPage openLastProductOfSection(String sectionName) {
         var lastProductOfSection = getLastProductTitleOfSection(sectionName);
 
         $x(format(GOODS_SECTION_TEMPLATE, sectionName))
-                .$x(format("./following-sibling::ul//a[contains(@title,'%s')]", lastProductOfSection))
+                .$x(format("./following-sibling::ul//a[contains(@title,\"%s\")]", lastProductOfSection))
```

Comment:
why did you change to \"%s\" ?

Reply:
if product's name consists apostrophe then my contains is failing